ycms/cms/urls/protected.py

```diff
@@ -30,4 +30,10 @@
             ]
         ),
     ),
+    path("ward/<int:pk>/", ward.WardView.as_view(), name="ward_detail"),
+    path(
+        "ward/<int:pk>/discharge-patient/<int:assignment_id>/<int:bed_id>/",
```

Comment:
```suggestion
        "discharge-patient/<int:assignment_id>/",
```
Since the other information is no longer needed (see above)